CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.3.0 / 2016-05-05
+* [CHANGE] Drop _count suffix for loaded_mappings metric (#41)
```

Comment:
nitpick again. Let's leave a newline between heading and list for consistency and readability. Use `<backtick>_count<backtick>` or `\_count` to not break syntax highlighting.